### ISSUE TITLE
Update deploy.sh for docker

### ISSUE THREAD

**Issue by chandrams** [kruize/kruize#21]
Update deploy.sh to run kruize in docker containers.  Routine docker() needs to be renamed to docker_container or as appropriate as it recursively gets called when docker commands are run from docker_prereq().

**Comment by dinogun** [comment]
The deploy.sh script does not yet have support for docker, will be shortly adding a PR for that.

**Comment by dinogun** [comment]
This is now available from PRs #22 and #23, you can use it as below
```
# Start kruize in docker mode
$ ./deploy.sh -c docker

# Terminate kruize in docker mode
$ ./deploy.sh -c docker -t
```
Documentation PR coming up later.
